src/components/qas/questions/qaCalculator/index.jsx

```diff
@@ -99,7 +99,21 @@ class QACalculator extends React.PureComponent {
     }));
   }
 
-  handleBlur = () => {
+  handleMinMax = target => {
+    const { min, max, value, dataset } = target;
+    const { valType } = dataset || {};
+    if (min && +min > +value) {
+      target.value = min;
+      this.handleChange(min, valType);
+    }
+    if (max && +value > +max) {
+      target.value = max;
+      this.handleChange(max, valType);
+    }
+  };
+
+  handleBlur = e => {
```

Comment:
So, I think all in all your handleMinMax is the right approach, however, I think to get the effect we're after you need to implement it in onChange not onBlur (see related comment above).  Why do I say that?  Well, if our goal is to never show the user unreasonable calculations then we can't "wait" until blur.  As is, the user can still enter any extreme number they want and they will see the resulting wild calculated values as long as the text input retains focus.  

Particularly because this is currently the only question on the page on which it is implemented, and the field we're constraining is the last field in the question, it is entirely reasonable to expect the input won't blur until the user advances to the next page (end of the investigation in this case but whatev).  So that means their answer could be updated without their even noticing or understanding why (n.b. I will ask Ardis to include language on this page about why we're constraining the observer distance value).  

I think this could be alleviated by acting upon the value entered immediately after it has been entered (in onChange); immediately before an answer is created and saved to state.